### PR TITLE
feat: support helper lookup for Storage.store

### PR DESCRIPTION
**Pull request by taybenlor**
Adds support for using the helper list as storage as well as loading.
Removes deprecated store from BuiltinHelper, this prevents it being
used as part of storage for the helper list.

### CODE REVIEW

**Comment by taybenlor** [comment]
@wryun can I get some eyes on this to see if it looks sane?

**Comment by wryun** [comment]
Sorry, missed this. Seems sane (at least it did when you explained it to me :) ), and worth PRing upstream when the stars align. Tests, wooh!

**Comment by taybenlor** [comment]
I think we may need to discuss it with the Scratch team before merging upstream. Once I started thinking about it, it doesn't *super* make sense to have saving prefer certain helpers to others, since you may end up storing it somewhere unexpected. However I think that suggests the whole system may need a bit more of a rethink. This works for us for now anyway.

**Comment by taybenlor** [comment]
@wryun this is ready for review now.

File: src/WebHelper.js [hunk 1]
```diff
@@ -99,6 +99,10 @@ class WebHelper extends Helper {
 
         let storeIndex = 0;
         const tryNextSource = () => {
+            if (storeIndex >= stores.length) {
```
Comment (wryun) [inline src/WebHelper.js:102]: not clear to me why this was added

**Review by wryun — APPROVED** [review]
LGTM. :ghost:

**Review by wryun — APPROVED** [review]
LGTM

**Pull request merged**